xbmc/epg/EpgInfoTag.cpp

```diff
@@ -488,6 +512,83 @@ std::string CEpgInfoTag::Plot(bool bOverrideParental /* = false */) const
   return retVal;
 }
 
+void CEpgInfoTag::SetOriginalTitle(const std::string &strOriginalTitle)
+{
+  CSingleLock lock(m_critSection);
+  m_bChanged |= (m_strOriginalTitle != strOriginalTitle);
+  m_strOriginalTitle = strOriginalTitle;
+}
+
+std::string CEpgInfoTag::OriginalTitle(bool bOverrideParental /* = false */) const
+{
+  std::string retVal;
+  CSingleLock lock(m_critSection);
+  if (bOverrideParental || !m_pvrChannel || !g_PVRManager.IsParentalLocked(*m_pvrChannel))
+    retVal = m_strOriginalTitle;
+  
+  return retVal;
+}
+
+void CEpgInfoTag::SetCast(const std::string &strCast)
+{
+  CSingleLock lock(m_critSection);
+  m_bChanged |= (m_strCast != strCast);
+  m_strCast = strCast;
+}
+
+std::string CEpgInfoTag::Cast(void) const
+{
+  return m_strCast;
```

Comment:
we change the channel pointers and recordings, but not the contents like this stuff. we only import new tags. and I guess we should refactor at some point soon to not need the setter for channel tags either, and const a bunch of methods